diff --git a/node_modules/react-native-otp-verify/android/build.gradle b/node_modules/react-native-otp-verify/android/build.gradle
index dc23f5d..6636a35 100644
--- a/node_modules/react-native-otp-verify/android/build.gradle
+++ b/node_modules/react-native-otp-verify/android/build.gradle
@@ -3,7 +3,6 @@ buildscript {
         repositories {
             google()
             mavenCentral()
-            jcenter()
         }
 
         dependencies {
@@ -50,7 +49,6 @@ repositories {
     }
     google()
     mavenCentral()
-    jcenter()
 }
 
 dependencies {
